charts/brig/values.yaml

```diff
@@ -100,6 +100,8 @@ config:
     # Disable one ore more API versions. Please make sure the configuration value is the same in all these charts:
     # brig, cannon, cargohold, galley, gundeck, proxy, spar.
     # setDisabledAPIVersions: [ v3 ]
+    setFederationStrategy: allowNone
```

Comment:
The old setting for cloud was:

```
    federationStrategy:
      allowedDomains: []
``` 

Should we set it to `allowDynamic` with an empty list here, too, to exactly match the old behavior?

Reply:
the two are different in behavior if the database table is non-empty.  so i'd argue `allowNone` is keeping behavior as unchanged as can be.